CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [1.42.86](https://github.com/brave/brave-browser/releases/tag/v1.42.86)
+
+ - Added Aurora EVM chain to preloaded chains for Brave Wallet. ([#23522](https://github.com/brave/brave-browser/issues/23522))
+ - Added the ability to debounce URLs based on a regex applied to the path. ([#23121](https://github.com/brave/brave-browser/issues/23121))
+ - Added pending request notification to Wallet's icon badge for Decrypt screen. ([#23613](https://github.com/brave/brave-browser/issues/23613))
+ - Improved Wallet UI on various screens to show the gradient color when selected. ([#23556](https://github.com/brave/brave-browser/issues/23556))
+ - Improved fingerprint farbling in aggressive mode. ([#24126](https://github.com/brave/brave-browser/issues/24126))
+ - Updated default search engine to Brave Search for new installations in certain regions. ([#23814](https://github.com/brave/brave-browser/issues/23814))
+ - Removed known Marketo email tracking parameters from URLs. ([#9018](https://github.com/brave/brave-browser/issues/9018))
+ - Removed P3A protobuf reporting. ([#23147](https://github.com/brave/brave-browser/issues/23147))
+ - Fixed incorrect token being set as default when selecting Buy/Send from token details page. ([#22574](http://github.com/brave/brave-browser/22574))
```

Comment:
two items:
1. Suggest updating this item to clarify it is for Brave Wallet: "Fixed incorrect token being set as default when selecting Buy/Send from Brave Wallet token details page" or similar wording
2. URL is incorrect, should be "https://github.com/brave/brave-browser/issues/22574" not "http://github.com/brave/brave-browser/22574"